README.md

```diff
@@ -35,9 +35,19 @@ pipx ensurepath
 
 Upgrade pipx with `brew update && brew upgrade pipx`.
 
+
 ### On Linux, install via pip (requires pip 19.0 or later)
 
+-  Ubuntu 23.04 or above
+
+```sh
+sudo apt install pipx
+pipx ensurepath
 ```
+
+-  Ubuntu 22.04 or below
```

Comment:
```suggestion
Or, and for other distributions, install via pip:
```
Installing via `pip` is still recommended, since new versions of `pipx` will be directly uploaded to PyPi and be pip-installable, in contrast to the version freeze in Ubuntu.
I'd tend to remove this line.

Reply:
See #982. Installing via pip may not work. Or maybe we can suggest users use pipx through zipapp?
```
curl -L https://github.com/pypa/pipx/releases/latest/download/pipx.pyz -o pipx.pyz
alias pipx="/path/to/python/interpreter pipx.pyz"
```

Reply:
Sueggesting the pyz is probably the best thing to do. Instead of `alias` (which needs to be added to rc) it’s probably better to suggest adding a script fix somewhere in PATH that does `exec ..../python ..../pipx.pyz`

Reply:
Where should the zipapp be stored? And it must be updated by hand, this should probably be included in the script fix. 

Also, I don't think it should be suggested to tweak the PATH (if that is what you want).

Reply:
It can be added to `~/.local/bin`. `pipx ensurepath` would add that to PATH.

Reply:
Do you think of a script approach like in #849 ?

Reply:
Maybe we can merge this PR first, and leave all these works in the follow-up PRs? :)

Reply:
FYI, `pipx.pyz` contains a shellbang header so it can be made executable directly.